packages/node/src/cli.js

```diff
@@ -285,6 +285,13 @@ const main = async (args = hideBin(process.argv)) => {
         global: true,
         group: BEHAVIOR_GROUP,
       },
+      'scuttle-global-this': {
+        describe: 'Enable scuttling of globalThis',
+        type: 'boolean',
+        global: true,
```

Comment:
I don't think this is correct. Scuttling only happens at runtime, therefore this option should be defined in the builder for the `run` command (and not have `global: true`)

Reply:
(somewhere around https://github.com/LavaMoat/LavaMoat/blob/b4c9c4e981637dc007df28acb2da49c3e6680388/packages/node/src/cli.js#L368)

Reply:
fixed 937e97f3b